### ISSUE TITLE
Github Enterprise Support

### ISSUE THREAD

**Issue by steveizzle** [AndreasAugustin/actions-template-sync#155]
**Is your feature request related to a problem? Please describe.**
I would really love to use this sync action on our enterprise github instance

**Describe the solution you'd like**
An Option would be nice to pass in the URL of the enterprise Instance

**Describe alternatives you've considered**
No alternatives considered

**Additional context**
        with:
          enterprise_url: github.myenterprise.ua
          source_repo_path: <owner/repo>

**Comment by AndreasAugustin** [comment]
Hi @steveizzle thanks for the request.

I'll have a look into it.

**Comment by AndreasAugustin** [comment]
@all-contributors please add @steveizzle  for ideas

**Comment by AndreasAugustin** [comment]
@steveizzle it is implemented and released with version `v0.3.0-draft`.

```yaml
with:
  hostname: github.myenterprise.ua
  source_repo_path: <owner/repo>
```

Hope this is sufficient.
Happy coding :rocket:

**Comment by steveizzle** [comment]
@AndreasAugustin wow and thx. I would have helped on that one, but that was fast ^^
